Sources/Compiler/IR/InstructionID.swift

```diff
@@ -1,30 +1,30 @@
-/// A stable identity denoting an instruction in a module.
+/// The stable identity of an instruction in its module.
 ///
 /// - SeeAlso: `InstructionIndex`
 public struct InstructionID: Hashable {
 
-  /// The index of the function in which the instruction resides.
+  /// The function containing this identity.
   public let function: Module.Functions.Index
 
-  /// The address of the block in which the instruction resides.
+  /// The block containing this identity.
   public let block: Function.Blocks.Address
 
-  /// The address of the instruction.
+  /// The identity of the instruction in its block.
   public let address: Block.Instructions.Address
 
-  /// Creates a path with the given properties.
+  /// Creates an identity with the given properties.
   public init(
-    function: Module.Functions.Index,
-    block: Function.Blocks.Address,
-    address: Block.Instructions.Address
+    _ function: Module.Functions.Index,
+    _ block: Function.Blocks.Address,
+    _ address: Block.Instructions.Address
   ) {
     self.function = function
     self.block = block
     self.address = address
   }
 
-  /// Creates an index with the given properties.
-  public init(block: Block.ID, address: Block.Instructions.Address) {
+  /// Creates an identity with the given properties.
```

Comment:
```suggestion
  /// Creates an instance with the given properties.
```

But I don't think this comment is accurate; it has no property of type `Block.ID`.

Reply:
`Block.ID` is just a bundle for the properties. I think that is fine. I really want a way to document member wise-like initializers without ceremony.